### PR TITLE
Fix homepage typos, misc updates

### PR DESCRIPTION
**Pull request by beets**
- Fix KG size
- Fix caption for water bivariate map
- Update links for scatter landing
- Hide buttons on mobile for stat var tool

### CODE REVIEW

**Review by shifucun — APPROVED** [review]

**Review by chejennifer — APPROVED** [review]

File: static/js/tools/scatter/info.tsx [hunk 1]
```diff
@@ -130,7 +146,8 @@ function Info(): JSX.Element {
           </a>
         </li>
         <li>
-          <b>Bachelor Degree Attained vs Females Per Capita</b> for counties in{" "}
+          <b>Bachelor&apos;s Degree Attainment vs. Female Population Per Capita</b>{" "}
```
Comment (pradh) [inline static/js/tools/scatter/info.tsx:149]: nit: s/Female/Women/ ?

File: static/js/tools/scatter/info.tsx [hunk 2]
```diff
@@ -42,7 +42,7 @@ function Info(): JSX.Element {
       <ul>
         <li>
           <b>
-            Prevalence Of Coronary Heart Disease vs Max Temperature Relative To
+            Prevalence Of Coronary Heart Disease vs. Max Temperature Relative To
             2006 RCP 4.5 Difference Relative To Base Date
```
Comment (pradh) [inline static/js/tools/scatter/info.tsx:46]: Below we call it "Projected temperature rise", should we call it the same?

Comment (pradh) [reply static/js/tools/scatter/info.tsx:46]: Ditto in Map

Comment (pradh) [reply static/js/tools/scatter/info.tsx:46]: image

File: server/templates/static/homepage.html [hunk 3]
```diff
@@ -270,7 +270,7 @@ <h3>Water</h3>
           <a href="{{ chart_link }}">
             <img class="img-thumbnail" src="/images/home-water_withdrawal_temp.png" />
           </a>
-          <small class="caption"><a href="{{ chart_link }}">Water withdrawal for irrigation vs. projected temperature rise in California</a> (source: USGS, NASA)</small>
+          <small class="caption"><a href="{{ chart_link }}">Water withdrawal for irrigation vs. projected temperature rise across US counties</a> (source: USGS, NASA)</small>
```
Comment (pradh) [inline server/templates/static/homepage.html:273]: While making name tweaks, could we add "Water" to  "Withdrawal rates" in https://datacommons.org/tools/timeline?

File: static/js/tools/scatter/info.tsx (hunk 2 above)
Comment (pradh) [reply static/js/tools/scatter/info.tsx:46]: another nit: s/Non-Biogenic Emission Source, Carbon Dioxide/Non-Biogenic Carbon Dioxide/ ?

**Review by pradh — APPROVED** [review]
Thanks for the fixes!

File: static/js/tools/scatter/info.tsx (hunk 1 above)
Comment (beets) [reply static/js/tools/scatter/info.tsx:149]: Population of Women Per Capita?
We do use the term "Female" in many other places ...

Comment (pradh) [reply static/js/tools/scatter/info.tsx:149]: +1

File: static/js/tools/scatter/info.tsx (hunk 2 above)
Comment (beets) [reply static/js/tools/scatter/info.tsx:46]: thanks for the catch - updated

File: server/templates/static/homepage.html (hunk 3 above)
Comment (beets) [reply server/templates/static/homepage.html:273]: done

File: static/js/tools/scatter/info.tsx (hunk 1 above)
Comment (beets) [reply static/js/tools/scatter/info.tsx:148]: i did not update this yet.. lmk what you think

Comment (pradh) [reply static/js/tools/scatter/info.tsx:148]: For consistency, we can leave it as female for now.

Comment (beets) [reply static/js/tools/scatter/info.tsx]: updated, ptal

Comment (pradh) [reply static/js/tools/scatter/info.tsx:148]: lg, thx

**Comment by beets** [comment]
thanks all for the review!

**Pull request merged**